common/powershell/hyperv/hyperv.go

```diff
@@ -213,10 +213,27 @@ New-VM -Name $vmName -Path $path -MemoryStartupBytes $memoryStartupBytes -NewVHD
 			return err
 		}
 
+		err = DisableAutomaticCheckpoints(vmName)
```

Comment:
not very concerned with this here, especially because it follows existing style in the file, but for functions that return a single error, I like to see:

```
if err = ...; err != nil {
    return err
}
```

because it keeps the logic local

Reply:
Should I change it throughout the file now or save it for a separate PR?

Reply:
If you want to change it in this PR, go for it, but I won't reject it if you don't change it